src/tools/client/parseDuration.ts

```diff
@@ -18,3 +18,22 @@ export const parseDuration = (time: number, t: TFunction): string => {
 
   return eta;
 };
+
+export const secondsFromTimeString = (time: string | undefined): number | undefined => {
+  if (!time) return undefined;
+  const lastChar = time[time.length - 1];
+  if (!isNaN(+lastChar)) return Number(time);
```

Comment:
I dislike this shorthand syntax because the flow is less readable. I would suggest to never leave out brackets. Do you agree?